cmd/check_perf.go

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Coinbase, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/coinbase/rosetta-cli/pkg/results"
+	t "github.com/coinbase/rosetta-cli/pkg/tester"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+)
+
+var (
+	checkPerfCmd = &cobra.Command{
+		Use:   "check:perf",
+		Short: "Benchmark performance of time-critical endpoints of Asset Issuer's Rosetta Implementation",
+		Long: `This command can be used to benchmark the performance of time critical methods for a rosetta server.
+This is useful for ensuring that there are no performance degradations in the rosetta-server.`,
```

Comment:
The branding for Rosetta that I've seen so far capitalizes the first letter of the name. Is there a particular reason why it's in lowercase here? Is 'rosetta' a reference to the type of server and not to the API?

Also please correct the Long value to the below strings:
```suggestion
		Long: `This command can be used to benchmark the performance of time-critical methods for a rosetta server.
This is useful for ensuring that there are no performance degradations in the rosetta-server.`,
```
Also not sure why `rosetta-server` is hyphenated there but not the sentence above? (It should not be hyphenated in either case.)